### ISSUE TITLE
Error: type mismatch: val termRowVec = new BDenseVector[Double](row(normalizedVS, termId).toArray)

### ISSUE THREAD

**Issue by dbl001** [sryza/aas#72]
```
def topTermsForTerm(normalizedVS: BDenseMatrix[Double], termId: Int): Seq[(Double, Int)] = {
    // Look up the row in VS corresponding to the given term ID.
    val termRowVec = new BDenseVector[Double](row(normalizedVS, termId).toArray)

    // Compute scores against every term
    val termScores = (normalizedVS * termRowVec).toArray.zipWithIndex

    // Find the terms with the highest scores
    termScores.sortBy(-_._1).take(10)
  }

<console>:193: error: type mismatch;
 found   : breeze.linalg.DenseMatrix[Double]
 required: org.apache.spark.mllib.linalg.distributed.RowMatrix
           val termRowVec = new BDenseVector[Double](row(normalizedVS, termId).toArray)
```

![screen shot 2016-07-23 at 9 02 52 pm](https://cloud.githubusercontent.com/assets/3105499/17081741/e3f0a002-5118-11e6-960b-03602d678def.png)

                                                         ^

**Comment by sryza** [comment]
Hi @dbl001, what version of Spark are you running?

**Comment by dbl001** [comment]
Welcome to
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /___/ .__/\_,_/_/ /_/\_\   version 1.6.2
      /_/

**Comment by sryza** [comment]
Do the issues you ran into in https://github.com/sryza/aas/issues/71 apply here as well?

**Comment by dbl001** [comment]
Yes.  

But, I can only run the code on smaller Wiki dumps: 
E.g. 
val pages = readFile("hdfs://52.8.16.250:/user/hduser/enwiki-latest-pages-articles1.xml-p000000010p000030302", sc)

-rw-r--r--   1 hduser supergroup   562561825 2016-07-30 08:13 /user/hduser/enwiki-latest-pages-articles1.xml-p000000010p000030302
-rw-r--r--   1 hduser supergroup 52927948149 2016-07-26 19:43 /user/hduser/wikidump.xml
   
The full Wiki dump gets all kinds of Spark errors.  Maybe it’s due to problems with my EC2 instance - which I built entirely from source.
I’ll try again on Amazon’s EMR 4.7.2 to see if it works.

In any case, I’m not gettng the problem I reported.  :-)
